data/avro-schema/src/main/resources/avro/net/corda/data/chunking/Chunk.avsc

```diff
@@ -35,15 +30,7 @@
     {
       "name": "data",
       "type": "bytes",
-      "doc": "the binary data fop the chunk. Zero data has a special meaning, signaling the last chunk in the sequence"
-    },
```

Comment:
Careful with this one -speak to Viktor? - but I *think* it's something to do with forcing a re-upload of a cpi